apps/financial-aid/web-veita/src/components/ProfileUnit/Unit.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { Text, Box, Link } from '@island.is/island-ui/core'
+
+import * as styles from './ProfileUnit.css'
+import cn from 'classnames'
+
+const LinkItem = (link?: string, onclick?: () => void, content?: string) => {
+  return (
+    link && (
+      <Link href={link} color="blue400" onClick={onclick}>
+        {content}
+      </Link>
+    )
+  )
+}
+
+const ButtonItem = (onclick?: () => void, content?: string) => {
+  return (
+    onclick && (
+      <button onClick={onclick} className={styles.button}>
+        {content}
+      </button>
+    )
+  )
+}
+
+const OtherItem = (other?: string) => {
+  return (
+    other && (
+      <Box background="blue100" borderRadius="large" padding={2} marginTop={1}>
+        <Text variant="small">
+          „<em>{other}</em>“
+        </Text>
+      </Box>
+    )
+  )
+}
+
+interface Props {
+  info: {
+    title: string
+    content?: string
+    link?: string
+    onclick?: () => void
+    other?: string
+  }[]
+  className?: string
+}
```

Comment:
I like this a lot, I've been thinking of the structure, should we have this in the component self or have it live next to it and we have a naming convention... Something like `getUnitContent.ts`, what do you think?